plugins/javaembed/javaembed.cpp

```diff
@@ -3099,6 +3099,13 @@ class JavaEmbedImportContext : public CInterfaceOf<IEmbedFunctionContext>
     }
     ~JavaEmbedImportContext()
     {
+        if (persistMode == persistThread)
+        {
+            assertex(classname.length());  // MORE - what does this imply?
```

Comment:
an assert in this destructor would terminate the program.  It should either be removed or improved.

Reply:
I will remove it